pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -66,6 +66,8 @@ const (
 	ZeroedThickDiskType       = "zeroedThick"
 	VolDir                    = "kubevols"
 	RoundTripperDefaultCount  = 3
+	DummyVMName               = "kubernetes-helper-vm"
```

Comment:
Is a race condition possible if 2 volumes are being created for the same VM name? Should the name by dynamically generated?